models/components/README.md

```diff
@@ -13,7 +13,21 @@ Now you must implement the [interfaces](versions/base/component.go) inside your
 
 In the case that you want to change how much access you want to give externally, you will need to edit the methods on the [`interfaces`](versions/base/component.go).
 
-*TODO: Future patches will enable 1) more component.yaml and 2) using multiple versions. More details to come then.*
+### Adding A New Version
+
+1. Create a folder `versions/X_Y_Z` where the `X_Y_Z` is the version in semver notation.
+1. Create your Component struct and any other structs that will implement the different interfaces inside [here](versions/base/component.go).
+1. Add another variable to represent your version inside of [versions/parse.go](versions/parse.go)
+1. Add a check in the switch-case block for your new version that is represented by the variable created in step 3.
+    1. Follow the same logic seen in the other versions inside the switch-case block.
+1. Add tests case fixtures with valid and invalid data for your version along with the other fixtures.
+1. Add those cases to the [versions/parse_test.go](versions/parse_test.go)
```

Comment:
😍 these instructions! Minor nit: be consistent about putting all of the paths in backticks.